unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -6795,51 +6794,25 @@ public bool objectIsCurrentlyVisible(SimObjPhysics sop, float maxDistance)
             float? maxDistance = null,
             int maxPoses = int.MaxValue  // works like infinity
         ) {
-            if (360 % rotateStepDegrees != 0) {
-                errorMessage = "360 % rotateStepDegrees must be 0, unless 'rotations: float[]' is overwritten.";
-                if (markActionFinished) {
-                    actionFinished(success: false);
-                }
-                return null;
+            if (360 % rotateStepDegrees != 0 && rotations != null) {
+                throw new InvalidOperationException("360 % rotateStepDegrees must be 0, unless 'rotations: float[]' is overwritten.");
```

Comment:
Unnecessary for now but in future PR's it would be great if you could add some more information to these error message using string interpolation. I.e.
```
throw new InvalidOperationException($"360 % rotateStepDegrees (360 % {rotateStepDegrees} != 0) must be 0, unless 'rotations: float[]' is overwritten.");
```

Reply:
Agreed. Though was an easy update, so I fixed it.